yktrace/src/sir.rs

```diff
@@ -105,23 +105,26 @@ lazy_static! {
 #[derive(Debug, Hash, Eq, PartialEq)]
 pub struct SirLoc {
     pub symbol_name: String,
-    pub bb_idx: u32
+    pub bb_idx: u32,
+    pub addr: u64,
```

Comment:
What is this address again? Does it need a comment?

Do we need it for all blocks? i.e. can it be an `Option`?

Reply:
We should only need this for `SirLoc`s where `bb_idx == 0`. So we can turn this into an `Option`.